nni/compression/experiment/config/compression.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import List, Type
+
+from torch.nn import Module
+
+from nni.experiment.config import ExperimentConfig
+from nni.experiment.config.base import ConfigBase
+from .pruner import PrunerConfig
+from .quantizer import QuantizerConfig
+
+__all__ = ['CompressionConfig', 'CompressionExperimentConfig']
```

Comment:
PEP8 suggests to put `__all__` before imports.
https://peps.python.org/pep-0008/#module-level-dunder-names

Reply:
move it before `import` and after `from __future__ import`